### DISCUSSION TITLE
Structuration environnement pédagogique sur les modèles ouverts (pour évènement)

### DISCUSSION THREAD

**Discussion by AbcSxyZ** [Open-Models/Brique#347] [Share an initiative]
Le 10 juin se tiens dans le tiers-lieu Ideavox (Genève) le [Festijovial](https://festijovial.org/), un festival culturel avec de la musique et différents ateliers, notamment un espace « hacking the school » pour permettre un débat citoyen sur l'éducation.

L'objectif est de construire tout un environnement pédagogique pour sensibiliser et faire de l'éducation aux modèles ouverts, avec une intention de venir nourrir les débats sur un sujet donné, ici l'école (plus d'infos sur [ce lien](https://ideavox.org/node/265)).

Il semble imaginable d'organiser un environnement pédagogique générique, modulaire et réplicable pour d'autres évènements, voire pour un cadre plus scolaire.

**Feuille de route :** https://annuel2.framapad.org/p/open-models-environnement-feuille-route-4738927419

## Zones pédagogiques

L'idée est de définir plusieurs espaces au sein de la structure dédiés chacun à l'exploration d'un (ou plusieurs) modèle ouvert, avec chaque zone qui permet en plus d'explorer au moins un sous-concept en lien, comme celui de commun numérique.

Chaque zone serait composée :
- d'un ensemble d'affichages, sur le modèle ouvert introduit et le sous-concept de la zone
- une activité à expérimenter, comme une exploration de ressources depuis un ordinateur
- d'une(/des) personne(s) qui anime(nt) l'espace, qui ont une maîtrise des concepts pour nourrir les échanges

Pour l'affichage, il y aurait :
- une fiche d'introduction au modèle ouvert de la zone
- des informations/documents sur le sous-concept de la zone
- des exemples du modèle ouvert

**Comment by slrleao** [comment]
Merci pour les infos !

Quelques questions suite à notre dernier call:

1. Est-ce que S. et moi on s'occupe uniquement de la présentation OKA et du coin open education ?
2. Est-ce que tu as aussi besoin de nous pour la partie open hardware ? Si jamais ça m'interesse aussi si besoin.
3. Comment rejoindre le hub d'ideavox ?
4. Quand est-ce que fait la prochaine réunion et qu'est-ce qu'on doit avoir d'ici là ?

**Reply by AbcSxyZ** [reply]
> Est-ce que S. et moi on s'occupe uniquement de la présentation OKA et du coin open education ? Est-ce que tu as aussi besoin de nous pour la partie open hardware ? Si jamais ça m'interesse aussi si besoin.

Au niveau du travail, il y a toutes les affiches à faire et c'est la mission principale, cela serait bien si on arrive à se répartir la création du contenu. D'ici la fin de semaine, j'aimerais au moins arriver à en finir le contenu pour une, ça permettra sûrement d'éclaircir pour le reste.

Trouver les personnes pour animer ces zones est plus secondaire.

> Comment rejoindre le hub d'ideavox ?

Vous pouvez créer un compte sur https://ideavox.org, L. doit valider ensuite.

> Quand est-ce que fait la prochaine réunion et qu'est-ce qu'on doit avoir d'ici là ?

Pour moi, il faut avancer sur ces contenus et commencer à les créer, et on voit comment on a besoin d'organiser des visio au fil de leur construction. Le travail se fait depuis la feuille de route, les sections sont par zone(/affiche) :  https://annuel2.framapad.org/p/open-models-environnement-feuille-route-4738927419

Pour le moment c'est plutôt de poser les idées et voir ce qu'il serait bien d'aborder, comment créer un modèle de base pour une fiche de présentation.

Perso, j'hésite encore à me lancer entre la fiche open software et open science

**Comment by AbcSxyZ** [comment]
J'ai avancé sur l'open software, je vous laisse voir directement le contenu sur la [feuille de route](https://annuel2.framapad.org/p/open-models-environnement-feuille-route-4738927419). Je vais discuter demain pour commencer à faire une première affiche avec ce que j'aurais déjà (tout risque de ne pas être complet).

Voir si on met d'autres sections, si il y aurait assez de contenu pour une A3 (ça serait le format de base) ou que sais-je. Pour le moment j'ai en tête de partir sur un squelette un peu comme ça :

![plan-fiche-modele-ouvert](https://github.com/Open-Models/Brique/assets/34010605/9e1fbe71-fe0f-4696-ba51-968f4dca5a0d)

Les sections de concepts permettraient de développer différents aspects, comme les (patrimoines) communs numériques, faire le lien avec l'environnement pédagogique dans l'open education, etc.

Si jamais je n'y ai pas pensé quand je t'ai répondu la dernière fois, mais tous les mardis on fait une réunion de travail, la prochaine étant demain : https://ideavox.org/node/296

**Comment by AbcSxyZ** [comment]
Le texte est à revoir et n'est plus bon, on a dû retirer du contenu pour le design (ce qui force à alléger les explications), mais voilà ce que ǎ donne **(les tailles de textes sont une vraie contrainte)** :
![open-models](https://github.com/Open-Models/Brique/assets/34010605/de80aa99-45fd-4b83-92cf-976148b80fd8)

**Reply by AbcSxyZ** [reply]
J'ai mis le fichier source (format svg), on l'a produit depuis Inkscape, j'ai juste un lien wetransfer pour le moment vu qu'il n'y a pas encore de dépôt pour ces contenus : https://we.tl/t-qJEfWoFT4o

**Reply by AbcSxyZ** [reply]
Des avancées pour une page open science ! Je vais attaquer l'open education directement ce week-end, j'aimerais bien arriver à finir tout ça d'ici mardi pour s'occuper plutôt de l'organisation ensuite.

![OPEN-MODELS_open-science](https://github.com/Open-Models/Brique/assets/34010605/21cc2514-a6c7-4125-8a31-967083b005b5)

**Reply by AbcSxyZ** [reply]
Et à peu près 3/4, des détails à régler genre orthographe, les qr codes, etc. mais ça commence à prendre bien forme.
![OPEN-MODELS_open-education](https://github.com/Open-Models/Brique/assets/34010605/88ca4e32-8a73-42ef-88b3-152e7ce7d2cf)

**Comment by AbcSxyZ** [comment]
**Mise à jour 03/06** 

On commence à avoir tout le contenu pour 3 affiches sur 4 : open software/science/education (voir précendents messages).

Des relectures et des affinages à faire, au moins sur l'orthographe ou les QR code, mais la base est là. 

**Pour la suite, je vois au moins les étapes suivantes à faire :** 
- Créer un dépôt pour tous ces documents, avec une documentation sur le fonctionnement de l'environnement pédagogique.
- Finir la fiche open hardware (@slrleao est en train de travailler dessus, à voir où tu en es ces prochains jours.)
- Voir pour une affiche ou qqch autour des licences creative commons, des droits d'auteurs
- Réfléchir pour les images des affiches
- Définir les personnes qui peuvent venir animer des zones, quand et comment.

Je pense qu'ils serait intéressant de créer quelques autres contenus, je pense notamment pour illustrer la notion d'archive ouverte, de patrimoine numérique commun avec le moteur CORE, j'imagine une forme de toile d'affiches d'archives de plusieurs institutions, qui se rejoignent vers le moteur de recherche et un ordinateur. Avec la possibilité de faire quelque chose de similaire pour l'open education.

Je vais voir où et comment publier ces contenus, je pense à GitHub, ça permettra d'avoir un environnement de travail un peu plus organisé pour finaliser cet environnement pédagogique.

Ça pourrait être cool si on finit ces 4 affiches et ce contenu un peu annexe d'ici mardi, pour pouvoir focus sur d'autres choses autour de l'organisation de « hacking the school ».

**Reply by AbcSxyZ** [reply]
Je suis en train de créer un dépôt GitHub pour l'environnement pédagogique : https://github.com/Open-Models/environnement-pedagogique

Cela va permettre d'organiser un peu mieux le travail et le ressources. J'ai réfléchi à la configuration à avoir, histoire de pouvoir collaborer et indexer les ressources.

J'en arrive à la conclusion d'un dépôt GitHub pour collaborer, avec une indexation depuis un lien donné à [OERCommons](https://www.oercommons.org/) qui sera à faire.

Ce n'était pas indispensable, mais j'ai choisi d'utiliser Zenodo pour archiver les versions, l'idée je pense est d'abord de rediriger les gens sur Zenodo, à partir d'un DOI. Ça permet d'avoir uniquement le contenu à récupérer sans l'interface plus intimidante de Github, en plus de profiter de quelques bénéfices de l'open science. Je teste aussi un peu pour voir l'intérêt, à réfléchir si conserver Zenodo est pertinent.

Pour créer des affiches, il y a petit guide de mofication, qui fait passer par Inkscape : https://github.com/Open-Models/environnement-pedagogique/blob/main/sources/guide-modification.md

**Reply by slrleao** [reply]
Merci, super travail !

J'ai terminé le contenu de l'open hardware et je termine la mise en page (je dois réduire un texte).

Je vais essayer de faire une affiche avec des images et quelques mots clés pour illustrer l'exemple de Prusa pour le hardware par exemple.

Je prend volontier la réalisation d'une autre affiche. L'économie de la contribution pour montrer comment toutes ces dynamiques peuvent s'inscrire dans un milieu économique ? Ou alors une de celles que tu as proposé.

On fait donc un point mardi de 14h à 16h sur le lien Ideavox ?

**Reply by slrleao** [reply]
Est-ce qu'il faut garder les sources pour les citer quelque part pour la publication ?

**Reply by AbcSxyZ** [reply]
Dès que tu as quelque chose à partager pour l'open hardware je suis preneur ! Je vais finaliser les affiches que j'ai demain, faire quelques retouches, correction orthographique, j'aimerais bien qu'on puisse imprimer ça mardi pour pouvoir travailler sur l'organisation du dôme.

**Il y a un truc sur lequel je réfléchis, de faire un réseau d'archive ouverte/bibliothèque(/forge) :**
![réseaux modèles ouverts(1)](https://github.com/Open-Models/Brique/assets/34010605/2c90ed41-2468-4757-b797-5d222b00d5f7)

C'est pour donner quelque chose de visuelles à cette exploration de ressources, illustrer la notion de patrimoine numérique commun. En montrant la variété d'archives ouvertes(/journaux/bibliothèques) et faire pointer le tout vers un moteur de recherche, qu'on place dans chaque zone. 

J'imagine ça comme un réseau d'affiches, relié par exemple par des ficelles. Des affiches simples avec le nom, logo, qr code du dépôt, quelques affiches un peu plus spéciales pour expliquer certains éléments comme les logiciels (invenio, HAL, core...). Réseaux d'affiches qui peuvent arriver sur un ordi à dispo pour utiliser ces moteurs de recherches, je pense en particulier avoir ça dans le dôme.


**J'aimerais vraiment bien arriver à en faire au moins 1 (open science), le reste dans la mesure du possible.**

Il y aurait d'autres choses qui me semblent importante comme sur les licences creatives commons/droits d'auteurs, sûrement d'autres détails, mais ça me paraît bien de travailler sur quelques comme ça. Est-ce que ça t'intéresse de t'orienter là-dessus ?

Ça marche pour mardi, mais est-ce que tu as éventuellement un moment ce lundi pour déjà discuter sur ça et la fiche open hardware ?

Lien visio : https://cal.com/open-models

**Reply by AbcSxyZ** [reply]
Pour les sources je ne savais pas trop, je les ais de côté mais pas dans le dépôt, on pourrait peut-être ajouter une bibliographie.

**Reply by slrleao** [reply]
![open-hardware-poster](https://github.com/Open-Models/Brique/assets/130675589/b0194b5c-222b-43a0-8583-88866e46f978)

**Reply by slrleao** [reply]
Oops on voit rien

**Reply by AbcSxyZ** [reply]
Tu trouves ? 

![fiche-open-hardware](https://github.com/Open-Models/Brique/assets/34010605/dc8bba7d-57d0-4bf4-b990-5fa68a2c8853)

Explications : https://github.com/Open-Models/environnement-pedagogique/pull/10

Je regarde ça ce début d'après-midi !

Edit : Je viens de voir que ce fix pose visiblement quelques problèmes d'alignement...

**Reply by AbcSxyZ** [reply]
Si jamais je viens d'envoyer mes correctifs pour les fiches science/software/education. Si tu veux faire une relecture et des retours.

J'ai encore quelques détails à faire autour type gérer les QR code, j'ajouterai bien un DOI aussi, et une version (0.1). En dehors de ça, pour moi ça commence à être valide.

**Reply by slrleao** [reply]
![open-hardware-poster](https://github.com/Open-Models/Brique/assets/130675589/79fc607a-6e83-43e9-86c2-f22940020956)

**Reply by slrleao** [reply]
Tu les as envoyé où ?

**Reply by AbcSxyZ** [reply]
Je travaille directement sur le dépôt Github : https://github.com/Open-Models/environnement-pedagogique

**Reply by AbcSxyZ** [reply]
DOI, version et QR codes fait. Si jamais le template que tu utilises est à changer, il faudra juste récupérer les textes et partir sur ces dernières versions.

**Reply by slrleao** [reply]
Voilà j'ai modifié le template.
Je n'ai pas l'accès pour déposer mon fichier sur GitHub
![open-hardware-poster](https://github.com/Open-Models/Brique/assets/130675589/f331015b-2129-4bb9-96b8-165e40eec2c6)

**Reply by AbcSxyZ** [reply]
Pull request 😉 glhf !

**Reply by slrleao** [reply]
Mmh comment ça ?

**Reply by AbcSxyZ** [reply]
fork, clone, (checkout,) add, commit, push, pull request 💪
![plongeon](https://github.com/Open-Models/Brique/assets/34010605/417923e2-000c-46db-adee-f3424a219b5a)

**Comment by slrleao** [comment]
Pour le schema sur l'open science j'ai quelque chose comme ça:

1. Core (moteur de recherche)
   
   Core est un moteur de recherche non-profit, qui fournit l'accès gratuit à une vaste quantité de papiers de recherches en open access.
   Core fournit plusieurs services tels que l'intégration du moteur de recherche et de ses données dans une infrastructure externe.
   La mission de Core est de collecter des papiers scientifiques et de garantir un accès libre à ces derniers en optimisant la publication, le managment et la validation du contenu.

1.1 Invenio

    Invenio est un logiciel open source permettant le dépôts de documents numériques en open science.
    Il fournit des outils pour l'integration et la gestion des ressources pour des institutions académiques.
    Invenio est originellement développé par le CERN et est entretenu et développé par sa communauté.

1.1.1 Zenodo

        Zenodo est une plateforme pour le dépôt ouvert de contenu en open science.
        La plateforme est géré par le CERN et fonctionne grâce au logiciel Invenio.
        Zenodo utilise le système de DOI afin de classer et d'identifier les documents.
